### ISSUE TITLE
improved support for non-microsoft community activity via discussions instead of issues

### ISSUE THREAD

**Issue by vigouredelaruse** [Azure/elastic-db-tools#213]
folks are reporting that modern sdk versions are unsupported by this library for their use case.

since 2015 there has been a corresponding bent in the issue library relating to a dearth of commits, and a handing off by microsoft.

thus it's likely that issues relating to library entropy/atrophy would be better served by (the github) discussion format, resulting in better filtering of issues vs discussions 

so really this is a request for enabling of discussions on this repo.
regards

**Comment by PeterLavreniuk** [comment]
Hello everyone! 
This is a great idea! Meanwhile a lot of "issues" looks like improvements or discussion threads.
Like you said the best way is to filter existing issues.
It will also be great to hear feedback from developers. Perhaps this library has more issues than we can find in the github repository.

**Comment by vigouredelaruse** [comment]
indeed. there are fundamental issues with the library at this point. for instance the split-merge tool last times i checked depended on classic azure vm deployment. if anybody's using it perhaps it's time to port split-merge to a function app or a logic app or derive guidance for why it's necessary or not. there's the business of formal support for .net core. and who knows what the mature end of a roadmap could look like. certainly a roadmap would be someplace to start. etc
